index.html

```diff
@@ -214,6 +216,7 @@ <h2>
           maxlength="30"
           required
         />
+        <label for="email" class="disnone">Email address </label>
         <input
```

Comment:
- [ ] Good job with this `input`  labelled here :+1: , keep doing the good work :+1: